bin/git-ignore-io

```diff
@@ -8,16 +8,26 @@ update_gi_list() {
 }
 
 print_in_alphabetical_order() {
-    for i in {a..z};
+    first=true
```

Comment:
IMHO, it is better to declare `local first ignorable first_character previous_first_character` before using those variables in function, to avoid polluting outer variable scope.